common/vcomputer-system.cpp

```diff
@@ -329,6 +360,11 @@ namespace tec {
 					Computer* computer = new Computer();
 					computer->In(comp);
 					ComputerComponentMap::Set(entity_id, computer);
+					for (auto device : computer->devices) {
+						if (device.second->device->DevType() == 0x03) { // 0x03 is keyboard DevType
```

Comment:
`DevType()` is one of a group of methods designed for the emulation to identify devices.
Type 0x03 is "HID", which in theory can include things that aren't keyboards.
Using it this way will certainly work for now, but it isn't future proof.

Reply:
I wasn't sure the best way to know if it should be able to receive focus. If there is a better way I will adjust accordingly.